doc/source/user_guide/offline/file_session.rst

```diff
@@ -64,39 +70,43 @@ Single-phase
     'y-component': 'SV_V',
     'z-component': 'SV_W'}}
    >>> transaction = file_session.fields.field_data.new_transaction()
-   >>> transaction.add_surfaces_request([3, 4])
-   >>> transaction.add_scalar_fields_request("SV_T", [3, 4])
-   >>> transaction.add_vector_fields_request("velocity", [3, 4])
-   >>> transaction.get_fields()
-   {(('type', 'scalar-field'),
-     ('dataLocation', 1),
-     ('boundaryValues',
-      False)): {3: {'SV_T': array([293.14999, 293.14999, 293.14999, ..., 293.14999, 293.14999,
-             293.14999])}, 4: {'SV_T': array([293.14999, 293.14999, 293.14999, ..., 293.14999, 293.14999,
-             293.14999])}},
-    (('type',
-      'vector-field'),): {3: {'velocity': array([0., 0., 0., ..., 0., 0., 0.]), 'vector-scale': array([0.1])},
-     4: {'velocity': array([ 3.32643010e-01,  6.64311343e-03,  0.00000000e+00, ...,
-              4.56052223e-01,  2.45034248e-01, -1.24726618e-15]),
-      'vector-scale': array([0.1])}},
-    (('type',
-      'surface-data'),): {3: {'faces': array([   4,    3,    2, ...,  727,  694, 3809], dtype=uint32), 'vertices': array([ 0.        , -0.1016    ,  0.        , ...,  0.00620755,
-             -0.19304685,  0.03033731])},
-     4: {'faces': array([   4,  295,  294, ...,  265, 1482, 2183], dtype=uint32),
-      'vertices': array([ 0.        , -0.1016    ,  0.        , ...,  0.06435075,
-             -0.08779959,  0.        ])}}
-   >>> from ansys.fluent.core.services.field_data import SurfaceDataType
-   >>> file_session.fields.field_data.get_surface_data([SurfaceDataType.Vertices], [3, 4])[3].shape
+   >>> surf_request = SurfaceFieldDataRequest(
+   >>>      data_types=[SurfaceDataType.Vertices, SurfaceDataType.FacesConnectivity],
+   >>>      surfaces=[3, 4],
+   >>> )
+   >>> sc_request = ScalarFieldDataRequest(field_name="SV_T", surfaces=[3, 4], node_value=False, boundary_value=False)
+   >>> vc_request = VectorFieldDataRequest(field_name="velocity", surfaces=[3, 4])
```

Comment:
@prmukherj These obscure names would be picked up by the tooling.

This issue is that review cycles are too lengthy.

Reply:
Done, Thank you.